src/styles/overlay-panel.less

```diff
@@ -16,6 +16,7 @@
   border: 1px solid @color-pf-black-400;
   height: 50%;
   left: 20px;
+  min-height: 350px;
```

Comment:
Why not just let the panel size for the content? Then small/large isn't needed at all, and panel would probably work better on mobile.

Then we can give the order service wizard a fixed height if needed so switching steps doesn't resize.